Changelog.md

```diff
@@ -1,3 +1,30 @@
+### 0.7.2 (2020-12-17)
+
+Features:
+
+* Applied the [Fixed-width SIMD](https://github.com/webassembly/simd) proposal.
+  * Please see the [document](https://github.com/second-state/SSVM/blob/master/doc/simd.md) for detail.
+* Supported toggle enable or disable different proposals.
+* Added the option to enable or disable compiling tools.
+
+Fixed issues:
+
+* Fixed `call_indirect` table index checking in the validation phase.
+
+Tools:
+
+* Deleted `ssvmr` tool.
+  * Please use the same tool `tools/ssvm/ssvm`.
+* Moved the tools in `tools/ssvm-aot` into `tools/ssvm`.
```

Comment:
* Combined the tools folder. All tools in `tools/ssvm-aot` are moved into `tools/ssvm` now.